data/reference/countries_regions.csv

```diff
@@ -203,7 +203,7 @@ RUS,Russia,RU,RUS,922.0,RUS,365.0,RUS,RU,USSR,RUS,RUS,http://www.wikidata.org/en
 RWA,Rwanda,RW,RWA,714.0,RWA,517.0,RWA,RW,RWND,RWA,RWA,http://www.wikidata.org/entity/Q1037,Rwanda,91.0,737.0,,
 BLM,Saint Barthlemy,BL,BLM,,,,,,,,,http://www.wikidata.org/entity/Q25362,Saint Barthélemy,297.0,795.0,,"[""Saint Barth\u00e9lemy""]"
 SHN,Saint Helena,SH,SHN,855.0,,,STH,XJ,,SHN,SHN,http://www.wikidata.org/entity/Q192184,"Saint Helena, Ascension and Tristan da Cunha",260.0,742.0,,"[""Saint Helena, Ascension and Tristan da Cunha"", ""St. Helena""]"
-KNA,Saint Kitts and Nevis,KN,KNA,361.0,SKN,60.0,STK,XD,,KNA,KNA,http://www.wikidata.org/entity/Q763,Saint Kitts and Nevis,227.0,672.0,,"[""St. Kitts and Nevis"", ""St.Kitts & Nevis""]"
+KNA,Saint Kitts and Nevis,KN,KNA,361.0,SKN,60.0,STK,XD,,KNA,KNA,http://www.wikidata.org/entity/Q763,Saint Kitts and Nevis,227.0,672.0,,"[""St. Kitts and Nevis"", ""St. Kitts-Nevis-Anguilla"", ""St.Kitts & Nevis""]"
```

Comment:
Seems like Anguilla is not part of Saint Kitts and Nevis these days, only historically.

https://en.wikipedia.org/wiki/Saint_Kitts_and_Nevis

Everything else looks good